scripts/run_backend_tests_test.py

```diff
@@ -916,8 +925,10 @@ def mock_subprocess_run(cmd: List[str], **_: str) -> MockProcessOutput:
             returned_output, coverage = run_backend_tests.check_coverage(
                 False, data_file=data_file)
 
-        self.assertEqual(returned_output, coverage_report_output)
-        self.assertEqual(coverage, 86)
+        expected_output = coverage_report_output
+        returned_res = returned_output
```

Comment:
Why rename this? Drop this and the line above and just use the original names.

Reply:
ok @seanlip i removed these lines.